docs/design/autoware-interfaces/components/vehicle-dimensions.md

```diff
@@ -4,7 +4,7 @@
 
 ![Vehicle Axes](images/vehicle_axes.svg){: style="width:500px"}
 
-The `base_link` frame is used very frequently throughout the stack, and is in the center of the rear axle.
+The `base_link` frame is used very frequently throughout the stack, and is a projection of the rear-axle center onto the ground surface.
```

Comment:
Although it's not directly related to this PR, I think the word `stack` can be changed to be more descriptive. Adding `Autoware` would be nice?

@xmfcx @LalithVipulananthan What do you think?

```suggestion
The `base_link` frame is used very frequently throughout the Autoware stack, and is a projection of the rear-axle center onto the ground surface.
```